graphene_sqlalchemy/utils.py

```diff
@@ -196,18 +196,23 @@ def __call__(self, *args, **kwargs):
         # No match, using default.
         return self.default(*args, **kwargs)
 
-    def register(self, matcher_function: Callable[[Any], bool]):
-        def grab_function_from_outside(f):
-            self.registry[matcher_function] = f
-            return self
+    def register(self, matcher_function: Callable[[Any], bool], func=None):
+        if func is None:
+            return lambda f: self.register(matcher_function, f)
+        self.registry[matcher_function] = func
+        return func
 
-        return grab_function_from_outside
+
+def value_equals(value: Any) -> Callable[[Any], bool]:
+    """A simple function that makes the equality based matcher functions for
+    SingleDispatchByMatchFunction prettier"""
+    return lambda x: (x == value or type(x) is value)
```

Comment:
Per default we now also check for isinstance. We should probably rename this to column_type_equals because it cannot be used for the builtins (see `str` + `ForwardRef`)